openstf/pipeline/train_model.py

```diff
@@ -158,11 +158,6 @@ def train_model_pipeline_core(
         except ValueError as e:
             logging.info("Could not compare to old model", exc_info=e)
 
-    # Do confidence interval determination
```

Comment:
Removed stdev generation here as it is also generated in the "train_model_pipeline_core"